### DISCUSSION TITLE
Star before adding as a list should be removed.

### DISCUSSION THREAD

**Discussion by udhaybegyall** [community/community#37393] [Other Feature Feedback, Questions, & Ideas]
What I expect lists to be collections of different repos in similar categories where I could find them easily.

I want to save the repository for later or if I like it but don't want to star it.

There would be many repositories that I would come across and want to save it for later so that I could look into it. But that doesn't mean I want to star every single repository.

So, we should be able to add the repo to list without giving it a ⭐. If I want to star it I can do that anytime.

**Comment by BayuAngora** [comment]
Star in Github is like a love button in Twitter. The purpose is not.just to save the content, but also to appreciate the man behind it at the same time.

**Reply by udhaybegyall** [reply]
I already know that, but there is also separate save button in those apps. Just like that I don't like to star every repo but want it there so later I don't forget about it. And later after checking it if I really liked it than I can start it.
